SPEC.md

```diff
@@ -0,0 +1,111 @@
+# adminbot spec
+
+This file documents both what adminbot does ambiently and what commands it responds to.
+
+## Behaviors
+
+### Welcome message
+
+adminbot knows what users it has seen and which ones it hasn't. When an unrecognized user joins
+`#tildetown`, adminbot welcomes them and gives some helpful hints.
+
+```
+--> | wren has joined
+<adminbot> Welcome, wren! I hope you're well. If you haven't seen it yet, please read over our wiki
+           page about this chat: https://tilde.town/wiki/socializing/irc
+```
+
+### Admin PSAs
+
+Admins have use of the `!psa` command which has adminbot anonymously relay a command to `#tildetown`.
+
+```
+<adminbot> ***** PSA: The server is being restarted in T-2 hours. Out your crushes. Hug your plushies.
```

Comment:
Perhaps this should do something more noticeable than just saying stuff in #tildetown - perhaps a wallops, or a /notice to everyone connected on IRC? It'd be easy to miss PSAs if you weren't online at the time otherwise.

Reply:
Also this looks like it's documented twice?